File: sqlUpdate.py

import pymysql
 #Open database connection

db = pymysql.connect(host="localhost", user="root", passwd="password", db="school")
#prep a cursor object using cursor() method; cursor is like a navigation method
db.autocommit(True)
cursor = db.cursor(pymysql.cursors.DictCursor)


sql = "UPDATE students SET age=16 WHERE name='jun'"
cursor.execute(sql)

cursor.close()
db.close()

File: sqlUpdate.php

<?php
$servername = "localhost";
$username = "root";
$password = "password";
$dbname = "school";

$conn = new mysqli($servername, $username, $password, $dbname);

// check connection
if($conn->connect_error){
    die("connection failed:".$conn->connect_error);
}

$sql = "UPDATE students SET name='Bao' WHERE name='Jun'";
$result =$conn->query($sql);

if ($conn->query($sql) === TRUE) {
    echo "Record updated successfully";
} else {
    echo "Error: " . $sql . "<br>" . $conn->error;
}

$conn->close();
?>
